### PR TITLE
e2e: fix test flakes where we specify a non-schedulable zone

### PR DESCRIPTION
**Pull request by justinsb**
If we specify a zone to the e2e tests, it will try to pre-provision
volumes into it for the inline tests.  We then will be unable to
attach a volume to test pods if we don't have (normal) nodes in that
zone.

Pick a zone that is schedulable for test pods.

### CODE REVIEW

File: pkg/apis/kops/v1alpha2/instancegroup.go [hunk 1]
```diff
@@ -49,6 +49,17 @@ type InstanceGroupList struct {
...
+	// InstanceGroupRoleMaster is a master role
+	InstanceGroupRoleMaster InstanceGroupRole = "Master"
+	// InstanceGroupRoleNode is a node role
+	InstanceGroupRoleNode InstanceGroupRole = "Node"
+	// InstanceGroupRoleBastion is a bastion role
+	InstanceGroupRoleBastion InstanceGroupRole = "Bastion"
+	// InstanceGroupRoleAPIServer is an API server role
+	InstanceGroupRoleAPIServer InstanceGroupRole = "APIServer"
+)
+
```
Comment (johngmyers) [inline pkg/apis/kops/v1alpha2/instancegroup.go:62]: What are these used for?

Comment (justinsb) [reply pkg/apis/kops/v1alpha2/instancegroup.go:62]: BTW I don't _really_ care too much here, if we don't want to duplicate the constants I can just import from the unversioned package.  I do think this is more correct though (even though it is more tedious).

Comment (johngmyers) [reply pkg/apis/kops/v1alpha2/instancegroup.go:62]: There doesn't appear to be a need to have versioned constants. The versioned structs are only ever accessed through serialization/deserialization and the apimachinery conversion code.

Comment (justinsb) [reply pkg/apis/kops/v1alpha2/instancegroup.go:62]: OK, I'll remove.

The side-question it opens up is whether we should be using the unversioned types.  For example, here we're parsing the output from `kops get cluster`, which is naturally versioned.  If we do need to convert it, should we convert to a versioned type?  (i.e. is the unversioned type actually just apimachinery internals "leaking").  But ... a topic for another day!

**Review by hakman — APPROVED** [review]

**Pull request merged**